src/transformers/integrations.py

```diff
@@ -1030,6 +1032,15 @@ def __init__(
             **neptune_run_kwargs
         }
 
+        self._should_upload_checkpoint = self._log_checkpoints is not None
+        self._recent_checkpoint_path = None
+
+        self._target_checkpoints_namespace = 'checkpoints'
```

Comment:
let's create `if else` instead of overriding.

Reply:
As you wish. Fixed :wink: